Bio/Align/__init__.py

```diff
@@ -159,21 +155,13 @@ def __init__(
         {'stats': 'CCCXCCC'}
         """
         if alphabet is not None:
-            if not isinstance(alphabet, (Alphabet.Alphabet, Alphabet.AlphabetEncoder)):
-                raise ValueError("Invalid alphabet argument")
-            self._alphabet = alphabet
-        else:
-            # Default while we add sequences, will take a consensus later
-            self._alphabet = Alphabet.single_letter_alphabet
+            raise ValueError(
+                "alphabet is no longer supported; its value should be None"
+            )
```

Comment:
Can we use the same message as in SeqIO, see #3121

Reply:
OK